lib/tasks/set_publish_status_for_package.rake

```diff
@@ -0,0 +1,17 @@
+# to run this rake task run the following command.
+# rake 'goodcity:set_publish_status_for_package[location_area, status]'
+# rake 'goodcity:set_publish_status_for_package[xyz, true]'
+
+namespace :goodcity do
+  task :set_publish_status_for_package, %i[location_area status] => [:environment] do |task, args|
+    location = Location.find_by(area: args.location_area)
+    if location&.packages&.any?
+      location.packages.update_all(allow_web_publish: true?(args.status))
+      puts "#{location.packages.count} packages updated to allow_web_publish: #{args.status}"
```

Comment:
Can you do the following as it will work for the Splunk logger - passing a key/value hash to the logger
```suggestion
      Rails.logger.info {task: "set_publish_status_for_package", msg: "#{location.packages.count} packages updated to allow_web_publish=#{args.status}"
```